pkg/apis/provisioning/v0alpha1/types.go

```diff
@@ -126,10 +126,13 @@ type RepositoryStatus struct {
 	// Initialized is true when the repository has been initialized
 	Initialized bool `json:"initialized"`
 
+	// The generation of the spec last time reconciliation ran
+	ObservedGeneration int64 `json:"observedGeneration"`
```

Comment:
+1 -- do we need "initialized" if we have this?

Reply:
Good question :) Not really anymore. Let me clean it in the other PR.